### PR TITLE
3920: Removed broken sites

### LINKED ISSUE #3920: Remove extractor for Viafree and more sites
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

_No response_

### Description

Hello.

I could not find anywhere else to report this

viafree.fi is redirecting to a payment service viaplay.fi

All these sites is redirecting to pluto.tv:
viafree.se
viafree.dk
viafree.no
viasat4play.no
tv3play.no
tv8play.se
tv6play.se
tv3play.se

tv6play.no is closed

### Verbose log

```shell
none
```

### PR DESCRIPTION
**Pull request by ivanskodje**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

Removed unsupported/broken sites:
viafree.se
viafree.dk
viafree.no
viasat4play.no
tv3play.no
tv8play.se
tv6play.se
tv3play.se
tv3play.dk
tv10play.se
play.nova.bg

Fixes #3920

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [x] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details><details><summary>Copilot Summary</summary>  

### Summary
🗑️🔥🧹

Remove obsolete and unused code for Viafree and other TVPlay sites. Update documentation and module imports accordingly.

> _Some extractors were no longer needed_
> _So from the codebase they were deleted_
> _`ViafreeIE` and more_
> _Were shown to the door_
> _And the docs and imports were updated_

### Walkthrough
* Remove Viafree support from documentation and extractor modules ([link](https://github.com/yt-dlp/yt-dlp/pull/7106/files?diff=unified&w=0#diff-b1159ce829de765d23840638f8df43d0c2130a15c9b1119cd772bcaea88d1467L1544), [link](https://github.com/yt-dlp/yt-dlp/pull/7106/files?diff=unified&w=0#diff-780b22dc7eb280f5a7b2bbf79aff17826de88ddcbf2fc1116ba19901827aa4e3L2082), [link](https://github.com/yt-dlp/yt-dlp/pull/7106/files?diff=unified&w=0#diff-7a738c978ec972abeb1dbb2b133e86a7bb7aa155902949b6fc5b0ff53c73b485L33-R33), [link](https://github.com/yt-dlp/yt-dlp/pull/7106/files?diff=unified&w=0#diff-7a738c978ec972abeb1dbb2b133e86a7bb7aa155902949b6fc5b0ff53c73b485L96-L206), [link](https://github.com/yt-dlp/yt-dlp/pull/7106/files?diff=unified&w=0#diff-7a738c978ec972abeb1dbb2b133e86a7bb7aa155902949b6fc5b0ff53c73b485L330-L426))

</details>

### CODE REVIEW

**Comment by Grub4K** [comment]
Why are they removed and not fixed instead?

**Comment by ivanskodje** [comment]
> Why are they removed and not fixed instead?

Pluto.tv have bought and taken over these URLs, and would be a new extractor and new task:
- viafree.se
- viafree.dk
- viafree.no

Does not exist anymore:
- viasat4play.no
- tv3play.no
- tv8play.se
- tv6play.se
- tv3play.se
- tv3play.dk
- tv10play.se

play.nova.bg already have an extractor at noveplay.py, and was removed from tvplay.py

File: supportedsites.md [hunk 1]
```diff
@@ -1541,7 +1541,6 @@
  - **VGTV**: VGTV, BTTV, FTV, Aftenposten and Aftonbladet
  - **vh1.com**
  - **vhx:embed**: [*vimeo*](## "netrc machine")
- - **Viafree**
```
Comment (pukkandan) [inline supportedsites.md:1544]: Revert. This will be auto-generated.

**Review by pukkandan — COMMENTED** [review]
> Pluto.tv have bought and taken over these URLs (already got an extractor plutotv.py):
> 
> * viafree.se
> * viafree.dk
> * viafree.no

Should these be added to plutotv's valid_url then?

File: supportedsites.md (hunk 1 above)
Comment (ivanskodje) [reply supportedsites.md]: Thank you for letting me know, I was not aware :upside_down_face:

**Comment by ivanskodje** [comment]
> > Pluto.tv have bought and taken over these URLs (already got an extractor plutotv.py):
> > 
> > * viafree.se
> > * viafree.dk
> > * viafree.no
> 
> Should these be added to plutotv's valid_url then?

Probably not, as the old URLs does not redirect anywhere (I am getting a 403). If you are using plutotv you would only work with plutotv URLs anyway and would have no need for supporting the old URLs.

**Review by pukkandan — APPROVED** [review]

**Pull request merged**